### PR TITLE
Initial implementation of generic library middleware

### LINKED ISSUE #429: Add official base library middleware
Use the taskcluster library as a base and import as `@neutrinojs/library`.

### PR DESCRIPTION
**Pull request by eliperelman**
This PR is waiting on scoped packages and fork middleware to land first. Fixes #429.

### CODE REVIEW

**Comment by eliperelman** [comment]
Relevant initial commit exists at https://github.com/mozilla-neutrino/neutrino-dev/pull/431/commits/2d26602308333a013cf61359cc388887e9c4896c.

File: packages/library/library/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "repository": "https://github.com/mozilla-neutrino/neutrino-dev/tree/master/packages/library",
+  "homepage": "https://neutrino.js.org",
+  "bugs": "https://github.com/mozilla-neutrino/neutrino-dev/issues",
+  "dependencies": {
+    "@neutrinojs/banner": "^7.3.0",
+    "@neutrinojs/clean": "^7.3.0",
+    "@neutrinojs/compile-loader": "^7.3.0",
+    "@neutrinojs/loader-merge": "^7.3.0",
+    "@neutrinojs/minify": "^7.3.0",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
```
Comment (edmorley) [inline packages/library/library/package.json]: I think `babel-plugin-dynamic-import-node` needs to be listed here too?

**Comment by edmorley** [comment]
It looks like the package is nested one level too deep in the repo? Perhaps an issue with the rebase?
`packages/library/library/...`

**Comment by eliperelman** [comment]
Addressing in #474.

**Pull request merged**